### PR TITLE
Plugins: Hide wp-admin Action Links if there's a Calypso UI

### PR DESCRIPTION
**Pull request by ockham**
Hide the wp-admin action links buttons (that point to wp-admin settings screens) if an extension's `package.json` sets `section.settings_path`.

As requested by @dbtlr in Slack. Sort of a follow-up to #14422; also related to p6q6po-9eH-p2#comment-87711.

Before:

image

After (hint: the 'Settings' button is gone):

image

To test:
* You need a JP site with the WP Super Cache plugin installed and active.
* Navigate to `calypso.localhost:3000/plugins/wp-super-cache/<yourJPsite>`.
* Verify that there's no 'Settings' button.
* You can also check other extensions that set `section.settings_path`, including WP Job Manager and Zoninator.
* Verify that the button is still visible for extensions that _don't_ set `section.settings_path` in their 
`package.json`. Right now, that's only WooCommerce (the button is labelled 'WP Admin' there).

### CODE REVIEW

**Comment by ockham** [comment]
Currently blocked by a very weird error:

image

**Comment by dbtlr** [comment]
Here's the Calypso live link that the bot didn't post :(

https://calypso.live/?branch=update/plugins-conditionally-hide-wp-admin-action-links

**Comment by ockham** [comment]
> So we're good with hiding all buttons here, regardless of whether or not they are for WP Admin settings. Is that right?

Yeah. I'd value UI sanity here over potentially missing some link to the author's website or whatever. Am I missing any other compelling cases?

File: client/my-sites/plugins/plugin-meta/index.jsx [hunk 1]
```diff
@@ -32,6 +31,7 @@ import PluginRemoveButton from 'my-sites/plugins/plugin-remove-button';
 import PluginInformation from 'my-sites/plugins/plugin-information';
 import WpcomPluginInstallButton from 'my-sites/plugins-wpcom/plugin-install-button';
 import PluginAutomatedTransfer from 'my-sites/plugins/plugin-automated-transfer';
+import { getExtensionSettingsPath } from 'my-sites/plugins/util';
```
[the remark this answers is not in the document]
Comment (ockham) [reply client/my-sites/plugins/plugin-meta/index.jsx]: He, I've seen both. For a while, I called mine `utils.js` until it dawned on me that the convention (also outside Calypso) is `util`. But it does seem like there are more `utils.js` than `util.js` files in Calypso, so I'll go with the majority 😄

**Comment by dbtlr** [comment]
I would say, at this point, given that we control the use case for all extensions that we shouldn't get too in the weeds about what links we show and don't show. We can always revise and change this later on if we want to get more specific.

**Pull request merged**